test/integration/rise-storage.html

```diff
@@ -354,25 +353,6 @@
           done();
         });
 
-        test("should retry storage request after rise-storage-api-error is encountered", function(done) {
```

Comment:
@donnapep @stulees actually I think these 2 tests should pass when rise cache is running. cheers